### DISCUSSION TITLE
Adding Searx (any iteration of it) as an engine in Chrome?

### DISCUSSION THREAD

**Discussion by Hebgbs** [searx/searx#2462] [General]
This baffles me — as an end-user of Firefox, I could add any variety of searx addresses available on Mycroft Project, but as an end-user of Chromium(-alike) browsers, I can't just right-click and add searx to my available search engines when viewing context menu options for the text input box.

**Why?** And if this isn't ever going to be resolved, could I at least have a template URL to add into my instance of Vivaldi? Because my baby-bottom smooth brain can't figure it out.

**Comment by Hebgbs** [comment]
Changing search method to GET worked — I got the option to add as search engine.

**Comment by unixfox** [comment]
Are you using a user agent randomizer or something? Because it works for me with a vanilla chromium browser.

**Reply by Hebgbs** [reply]
Not that I know of, I just use Vivaldi. Maybe searx defaults to GET for you? Or, current Chromium can deal with engines using POST whereas Vivaldi doesn't.
